edx_solutions_api_integration/users/views.py

```diff
@@ -750,6 +753,19 @@ def post(self, request, user_id):  # pylint: disable=R0915
             response_data['username'] = existing_user.username
             existing_user.save()
 
+            if old_username != username:
+                storage = get_profile_image_storage()
+                profile_image_names = get_profile_image_names(old_username)
+                new_profile_image_names = get_profile_image_names(username)
+                for old_image_size, old_image_name in profile_image_names.items():
+                    if storage.exists(old_image_name):
+                        old_image_path = storage.path(old_image_name)
+                        new_image_path = storage.location + '/' + new_profile_image_names[old_image_size]
```

Comment:
Please use os.path.join to concatenate paths. 
```
new_image_path = os.path.join(storage.location, new_image_name)
```

Reply:
done